README.md

```diff
@@ -30,17 +30,15 @@ To learn how to edit and build this repo's content, please refer to
 
 ## Versions and releases
 
-Istio maintains three variations of its public site.
+Istio maintains two variations of its public site.
```

Comment:
Why two? There are 3 bullets here.

Reply:
There are only two now: istio.io and preliminary.istio.io. The archive.istio.io won't be used for archive any more and instead we'd use istio.io/archive.

Reply:
ah ok, thanks